src/gcp_scanner/crawler/interface_crawler.py

```diff
@@ -26,6 +26,12 @@ class ICrawler(metaclass=ABCMeta):
 
   """
 
+  """Variable to identify if config file is needed
+
+  Access Type: Private
+  """
+  _config_depndency = False
```

Comment:
NIT: depndency/dependency. Here and everywhere else.

Reply:
Apologies, for the the Typo. Have fixed all its occurrences.